### ISSUE TITLE
Example with EC2 spot instances

### ISSUE THREAD

**Issue by matthewcummings** [terraform-aws-modules/terraform-aws-ecs#77]
## Is your request related to a new offering from AWS?
Is this functionality available in the [AWS provider for Terraform](https://registry.terraform.io/providers/hashicorp/aws/latest/docs)? See [CHANGELOG.md](https://github.com/hashicorp/terraform-provider-aws/blob/main/CHANGELOG.md), too.
  - No 🛑: please wait to file a request until the functionality is avaialble in the AWS provider
  - Yes ✅: please list the AWS provider version which introduced this functionality

## Is your request related to a problem? Please describe.

This module is great, just wondering if you could provide an example with EC2 spot instances, in addition to the Fargate spot examples.

## Describe the solution you'd like.

Basically, what I just described - an example with EC2 spot instances.

**Comment by antonbabenko** [comment]
This issue has been resolved in version 5.0.1 :tada:
